hack/jenkins/release_github_page.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script generates the Github Release page and uploads all the binaries/etc to that page
+# This is intended to be run on a new release tag in order to generate the github release page for that release
+
+# The script expects the following env variabls:
+# VERSION_MAJOR: The the major version of the tag to be released.
+# VERSION_MINOR: The the minor version of the tag to be released.
+# VERSION_BUILD: The the build version of the tag to be released.
+# ISO_SHA256: The sha 256 of the minikube-iso for the current release
+# GITHUB_TOKEN: The Github API access token. Injected by the Jenkins credential provider. 
+
+set -e 
+export TAGNAME=v${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_BUILD}
+export DEB_VERSION=${VERSION_MAJOR}.${VERSION_MINOR}-${VERSION_BUILD}
+
+export GITHUB_ORGANIZATION="kubernetes"
+export GITHUB_REPO="minikube"
+export PROJECT_NAME="minikube"
+export DARWIN_SHA256=$(cat out/minikube-darwin-amd64.sha256)
+export LINUX_SHA256=$(cat out/minikube-linux-amd64.sha256)
+export WINDOWS_SHA256=$(cat out/minikube-windows-amd64.exe.sha256)
+
+# Description could be moved into file on machine or fetched via URL.  Doing this for now as it is the simplest, portable solution.
+# ================================================================================
+export DESCRIPTION="# Minikube ${TAGNAME}
```

Comment:
(discussed offline, here for reference) Yes definitely, but I will wait until a future PR as I am going to sync the files first and it can't be broken out separately in Jenkins now.